beetsplug/beatport.py

```diff
@@ -237,18 +229,16 @@ def __init__(self, data):
         if "category" in data:
             self.category = data["category"]
         if "slug" in data:
-            self.url = "https://beatport.com/release/{}/{}".format(
-                data["slug"], data["id"]
+            self.url = (
+                f"https://beatport.com/release/{data['slug']}/{data['id']}"
```

Comment:
URLs are actually the one place where I think using `format` might be better than f-strings. It lets you see the structure of the URL more clearly than having the variables embedded. On the fence about this part.

Reply:
That's fair.  I guess I'll change them so that each parameter to the URL is named.  I also remember I saw a different way of implementing this somewhere in the codebase, more similar to `pathlib`'s `a / b` syntax; I'll see if I can find it again.